src/garage/tf/algos/dqn.py

```diff
@@ -185,8 +184,10 @@ def train_once(self, itr, paths):
         self.episode_rewards.extend(paths['undiscounted_returns'])
         last_average_return = np.mean(self.episode_rewards)
         for train_itr in range(self.n_train_steps):
-            if (self.replay_buffer.n_transitions_stored >=
-                    self.min_buffer_size):
+            # yapf: disable
```

Comment:
it seems like YAPF is misbehaving for comparison or logical operators?

Reply:
It works fine at some places for logical operators like https://github.com/rlworkgroup/garage/pull/773/files#diff-eec0e5bc5f0646f0458d535037a7bcf0R363 & https://github.com/rlworkgroup/garage/pull/773/files#diff-eec0e5bc5f0646f0458d535037a7bcf0R363
but goes crazy at others. And yapf doesn't have a flag that covers comparison operators at all

Reply:
It also messed up around some arithmetic operators sometimes like if the an arithmetic expression is passed to a function call such as [here](https://github.com/rlworkgroup/garage/pull/773/files#diff-bb9dfa32db8aec7c259bc87d036131adR53)

Reply:
i think comparison operators are considered logical operators -- but yes every time i've tried this, YAPF has had terrible W504 support.

you may also want to try `split_before_bitwise_operator`

you can increase `split_penalty_arithmetic_operator`, `split_penalty_bitwise_operator` and `split_penalty_bitwise_operator` if that doesn't work.  the defaults are here: https://github.com/google/yapf/blob/45cf63a4538a2d5790db5182b84aca8c1b3bc491/yapf/yapflib/style.py#L341 -- just try doubling them until it behaves correctly.

Reply:
[yapf source](https://github.com/google/yapf/blob/27fe4ad04fd61979ae8b734d13a569a29a3b79bd/yapf/yapflib/unwrapped_line.py#L520) has only 'and' and 'or' as logical operators. I don't see comparison operators mentioned at all.

Increasing the penalties helps 1 or 2 cases of logical operators in terms of line break but still not the best for readability. It largely does no good for arithmetic operators, and instead increasing it too much for arithmetic operators causes unexpected changes. 

TLDR: even with all these knobs and high penalties, yapf doesn't produce expected output